packages/containers/src/ComponentForm/ComponentForm.component.js

```diff
@@ -102,9 +102,9 @@ export class TCompForm extends React.Component {
 	}
 
 	componentDidUpdate(prevProps) {
-		const nextProperties = this.props.state.get('properties', Map());
```

Comment:
extracting the allocation outside of the function would also work and bring less noise isn't it?

Reply:
what ?